source/making_payments/index.html.md.erb

```diff
@@ -5,109 +5,137 @@ weight: 70
 
-* create a secure random ID (such as a UUID) and include this as part of the
-``return_url``, using a different ``return_url`` for each payment. Since a
-securely generated UUID is not guessable, fraudsters will not be able to
-intercept users’ payments.
+You can match users with payments with an encrypted cookie, or by creating a
+secure random ID:
 
->Note: If you create an ID yourself, you’ll likely need to store this in a
-datastore mapped to the payment ID just after you create a payment.
+### Use an encrypted cookie
+
+You can use an encryped cookie containing the `paymentId` from GOV.UK Pay. Your
+service should issue this when a payment is created, before sending the user
+to `next_url`. Users will not be able to decrypt an encrypted cookie, so a
+malicious user could not alter the payment ID and intercept other users’
+payments.
+
+### Create a secure random ID
+
+You can create a secure random ID, such as a UUID, and include this as part of
+the `return_url`. You should use a different `return_url` for each payment.
+Malicious users will be unable to guess securely generated UUIDs,
+
+If you use this method, you should store your IDs safely. For example, in a
+datastore mapped to the payment ID just after a payment is created.
 
 ## Accepting returning users
 
- A user directed to the return URL could have:
+ A user directed to the `return_url` could have:
 
  * paid successfully
 
- * not paid because their card was rejected or they clicked cancel
+ * not paid because their card was rejected or they selected cancel
 
  * not paid because your service (via the API) cancelled the payment in progress
 
  * not paid because of a technical error
 
 Your service should use the API to check the payment status when the user
-reaches the return URL, and provide an appropriate response based on the final
+reaches the `return_url`, and provide an appropriate response based on the final
 status of the payment attempt.
 
 ## When a user does not complete their payment journey
 
-The user may close their browser or lose internet connection in the middle of
-the payment flow on GOV.UK Pay. These users will not be redirected back to
-your service.
-
-You can still check on the status of these payments by making a GET request
-using the Location Header or Self Link, the same way you would if they were
-redirected, but just after a set time (eg, two hours).
+Users may close their browser or lose internet connectivity in the middle of
+their payment journey on GOV.UK Pay. In this event, users will not be
+redirected back to your service.
 
->Note: GOV.UK Pay will expire incomplete payments after 90 minutes, but you should expect an occasional success or failure if the user experienced problems right at the moment of the redirect.
+You can still [check the status of these
+payments](/payment_flow/#check-the-status-of-a-payment), but GOV.UK Pay will
+expire payments after 90 minutes. You can only be certain of the payment’s
```

Comment:
Hmm, I preferred it before.

Maybe it come something like this: “You can still [check the status of these payments](/payment_flow/#check-the-status-of-a-payment), but be aware that GOV.UK Pay will allow a user 90 minutes to complete their payment before it expires. You can only be certain of the payment’s final status after this time.”